php-worker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+#--------------------------------------------------------------------------
+# Image Setup
+#--------------------------------------------------------------------------
+#
+# To take a look at the'php-worker' base Image, visit its DockerHub page
+#    https://hub.docker.com/r/nielsvdoorn/laravel-supervisor/
+#
+
+FROM nielsvdoorn/laravel-supervisor
+
+#
+#--------------------------------------------------------------------------
+# Optional Supervisord Configuration
+#--------------------------------------------------------------------------
+#
+# Modify the ./supervisor.conf file to match your App's requirements.
+# Make sure you rebuild your container with every change.
+#
+COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+
+#
+#--------------------------------------------------------------------------
+# Optional Software's Installation
+#--------------------------------------------------------------------------
+#
+# If you need to modify this image, feel free to do it right here.
+#
+	# -- Your awesome modifications go here -- #
+
+
+#
+#--------------------------------------------------------------------------
+# Final Touch
+#--------------------------------------------------------------------------
+#
+
+RUN rm -r /var/lib/apt/lists/*
+WORKDIR /etc/supervisor/conf.d/supervisord.conf
```

Comment:
Should be at least `WORKDIR /etc/supervisor/conf.d/` since existed variant is not working:
```
ERROR: for php-worker  Cannot start service php-worker: oci runtime error: container_linux.go:247: starting container pr
ocess caused "chdir to cwd (\"/etc/supervisor/conf.d/supervisord.conf\") set in config.json failed: not a directory"
: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the ex
pected type
```